src/main/java/uk/gov/hmcts/reform/data/ingestion/configuration/StorageCredentials.java

```diff
@@ -11,10 +13,18 @@
     @Autowired
     AzureBlobConfig azureBlobConfig;
 
+    @Autowired
+    @Qualifier("credsreg")
+    com.microsoft.azure.storage.StorageCredentials storageCredentials;
+
```

Comment:
import the package on the top com.microsoft.azure.storage.

Reply:
We have same of the class kept from start. hence Package would need to be explicitly specified